randovania/games/samus_returns/json_data/Surface - East.txt

```diff
@@ -470,7 +472,7 @@ Extra - asset_id: collision_camera_011
       Morph Ball
   > Door to Charge Beam Shaft
       Morph Ball
-  > Right of Statue
+  > Start Point - Right of Statue
```

Comment:
They're not helper nodes: they only connction they have is thelmseves <-> nearest dock. Thus to avoid confusion as to why it leads to nowhere, i marked them with start point, as all other start points do the same.

Reply:
Ah whoops. Might be worth changing `Start` to `Spawn` to be even more clear

Reply:
Will do in a future PR, as literally all of them are called "Start Point".